wiki/Modifying-Your-Game-with-Scripts.md

```diff
@@ -0,0 +1,71 @@
+- have a Javascript file you want to add to your world or module, and   
+- edit the world or module's manifest to point to your Javscript code.
+
+Your Javscript file will usually live within the directory for the world or module in which it is used, but in reality, it could live anywhere in your Foundry userdata folder (see [Where Is My Data Stored?](https://foundryvtt.com/article/configuration/#where-user-data) on the Foundry KB for more information).  
+
+This guide assumes that your Javscript is in the root of the world's directory (i.e. `Data/worlds/my-world/my-script.js`), and will be adding the script to your World.  
+If you would instead like to add the script to a module, you can follow the guide below, replacing any use of "world" with "module", assuming your Javascript is in the root folder of the module's directory (i.e. `Data/modules/my-module/my-script.js`).
+
+# Adding a script to your manifest
+To include your Javascript file in your world:
+
+1. Navigate to your world's directory in your user data folder.
+2. Open world.json in a text editor (Visual Studio Code is a good choice, but almost any editor will do).
+3. In the world.json file, look for a line with the esmodules key. If there isn't one, add it after any of the existing keys, like this:
+```json
+{
+  "title": "My Cool DnD Game",
+  "esmodules": [],
+  "version": "1.0.0",
+...
+}
+```
+4. Add the path to your Javascript file to the esmodules array. If your Javascript file is stored in the root of the world directory as suggested above, just write the filename in quotation marks. It will look like this when you're done: `"esmodules": [ "my-script.js" ],` (note the comma is required at the end of the line, unless you've added this as the very last key in the JSON file).
+5. Save and close world.json.
+6. If the world was already active, return to setup and launch it again. Whenever you change the world.json, you must re-launch the world for the changes to take effect.
+
+
+# Examples
+Scripts rely on Hooks to automatically execute, see [Hooks](Hooks.md) for system specific Hooks that you can use, or the [Foundry API](https://foundryvtt.com/api/) for Hooks provided by the Core Foundry Software.
+Need Help? Feel free to jump in the Foundry VTT discord's #dnd5e or #macro-polo channels for assistance from the community. 
+
+## Adding an additional custom damage type
+```js
+// Adds a new "plasma" damage type that will be selectable as a new type of damage for weapons
+// and a new type of resistance for actors.
+// Note that this *might* not play well with modules which auto apply damage and resistances.
+Hooks.on("setup", () => {
+  CONFIG.DND5E.damageTypes.plasma = "Plasma";
+  CONFIG.DND5E.damageResistanceTypes.plasma = "Plasma";
+});
+```
+
+## Changing the Max Level Cap
+```js
+/// Increases the Maximum Level for a Player Actor to 30
+Hooks.on("setup", () => {
+  CONFIG.DND5E.maxLevel = 30;
+});
+```
+
+## Adding a New Weapon Type  
+```js
+

```

Comment:
Let's add some placeholder here so readers don't think it's an error.
```suggestion
  // TODO
```

Reply:
replaced